cypress/e2e/accessibility/07_accessibility.cy.js

```diff
@@ -0,0 +1,12 @@
+const urls = require('../sitemap.json'); // see sitemap.sh
+
+describe('Entire Website accessibility', () => {
+  urls.forEach((url) => {
+    it(`${url.length === 0 ? '/' : url}`, () => {
+      // Wait for page to load ...
+      cy.intercept(url).as('getPage');
+      cy.visit(url);
+      cy.checkPageA11y(url);    
```

Comment:
You want to use a `wait` with the @name of the intercept you are doing, like this:
```
      cy.intercept(url).as('getPage');
      cy.visit(url);
      cy.wait('@getPage').then(() => {
        cy.checkPageA11y(url);
      });
```
This ensures that the page is completely loaded before you run your checks.

Reply:
Perfect! I'll make that change.